html/webSocket.js

```diff
@@ -0,0 +1,36 @@
+const webSocketStatusElement = document.getElementById("webSocketStatus");
+const webSocket = new WebSocket("ws://" + window.location.host + "/ws");
+
+// Web socket open
+webSocket.onopen = function () {
+    webSocketStatusElement.innerHTML = "Web socket status: <b>Connected</b>";
+};
+
+// Web socket receive message
+webSocket.onmessage = function (e) {
+    let data = JSON.parse(e.data);
+    console.log("Message received:", data);
+    data.forEach(d => {
+       submitModalNewCommand(d.Title, d.Description, d.Command, d.Output)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['submitModalNewCommand' is not defined.](https://app.codacy.com/manual/ttimt/Short_Cutteer/pullRequest?prid=4845233)